File: challenges/4-fizz-buzz/python/emptybrain/fizzbuzz.py
value = input("Insira algum valor: ")
fizzbuzz: list = ["Fizz", "Buzz", "FizzBuzz"]
for output in range(1, int(value) + 1):
    if (output % 5 == 0) and (output % 3 == 0): 
        print(fizzbuzz[2])
        continue
    elif output % 3 == 0:
        print(fizzbuzz[0])
        continue
    elif output % 5 == 0:
        print(fizzbuzz[1])
    print(output)

File: challenges/4-fizz-buzz/java/izepon/FizzBuzz.java
import java.util.Scanner;

public class FizzBuzz {

	public static void main(String[] args) {
		
	   Scanner s = new Scanner(System.in);

	      System.out.println("Digite aqui o valor: ");
	      
	      int num = s.nextInt();

	      for (int i = 1; i <= num; i++) {

	         if(i % 3 == 0 && i % 5 == 0){
	            System.out.println("FizzBuzz");
	         }else if(i % 3 == 0){
	            System.out.println("Fizz");
	         }else if(i % 5 == 0){
	            System.out.println("Buzz");
	         }else {
	            System.out.println(i);
	         }
	      }
   }
}
